package messages;

message InertialState
{
    // Raw accelerometer data.
    optional float acc_x = 1;
    optional float acc_y = 2;
    optional float acc_z = 3;

    // Raw gyrometer data.
    optional float gyr_x = 4;
    optional float gyr_y = 5;
    optional float gyr_z = 6;

    // Filtered angle data.
    optional float angle_x = 7;
    optional float angle_y = 8;   
    optional float angle_z = 9;   
}
